src/app/products/loan-products/wizard/loan-product-wizard.component.scss

```
@@ -299,3 +299,76 @@ mat-checkbox {
+ * `.field-grid`; because an attribute selector outranks an element selector, it also beat the
+ * heading rule below and left section headings sharing a row with the first field while every
+ * checkbox stretched into its own column. Restyling the type is enough to make the two flows read
+ * alike, and it cannot break a layout it does not touch.
+ */
+.wizard-classic-step,
+.wizard-classic-review {
+  display: block;
+
+  /* Classic's section headings are page-level (h2/h3/h4); inside a wizard step they read as
+     oversized field labels unless toned down to the shell's eyebrow treatment. */
+  ::ng-deep h2,
```

Stylelint (17.14.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)